src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let robotCounter = 0;
+
+  return function(firstAction) {
+    return function(secondAction) {
```

Comment:
The parameter name 'secondAction' is not appropriate, consider renaming it to 'num2' or 'secondNumber'